### ISSUE TITLE
hubert 提取音频，已经执行完成了print(hubert_hidden.detach().numpy().shape)，但是程序一直没有正常结束

### ISSUE THREAD

**Issue by songpengshishen** [anliyuan/Ultralight-Digital-Human#139]
在第一步提取音频特征时：get_audio_feature，选择了hubert.py，已经执行完了并输出了 : print(hubert_hidden.detach().numpy().shape)
，但是程序没有正常结束退出，一直卡在哪里不动

**Comment by anliyuan** [comment]
不确定 可能是transformers版本的问题 更新下试试
